src/runtime/storageNG/storage-proxy.ts

```diff
@@ -34,10 +38,19 @@ export class StorageProxy<T extends CRDTTypeRecord> {
   constructor(
       crdt: CRDTModel<T>,
       store: ActiveStore<T>,
-      scheduler: StorageProxyScheduler<T>) {
+      apiChannelId: string,
```

Comment:
it's possible (per the way the api-channel works) that this will need to be the first argument in the constructor.

Reply:
moved to first place